airflow/utils/python_virtualenv.py

```diff
@@ -149,3 +157,74 @@ def write_python_script(
         )
     template = template_env.get_template("python_virtualenv_script.jinja2")
     template.stream(**jinja_context).dump(filename)
+
+
+def context_to_json(context: Context) -> str:
+    from airflow.models.param import ParamsDict
+    from airflow.models.taskinstance import SimpleTaskInstance
+    from airflow.serialization.serialized_objects import SerializedBaseOperator, SerializedDAG
+    from airflow.utils.context import Context
+
+    context_copy: dict[str, Any] = {}
```

Comment:
If it's only for fixing a mypy issue - you could add the type hint when you assigning `context_copy` below (L183)

Reply:
I had used it before, but had forgotten that I hadn't cleared it. I've fixed it, thanks.